integration/app/test-demo/src/pages/Menu.tsx

```diff
@@ -0,0 +1,18 @@
+import { Heading } from "amazon-chime-sdk-component-library-react";
+import React from "react";
+import { Link } from "react-router-dom";
+import routes from "../constants/routes";
+
+const Menu: React.FC = () => (
+  <>
+    <Heading level={4}>
+      React Component Library Test Demo
+    </Heading>
+    <nav>
+      <ul>
+        <li><Link id="simple-app" to={routes.SIMPLE_APP}>Simple App</Link></li>
```

Comment:
Can we navigate to the specific test right away without going to this page? In an actual test, we probably want to skip this page.

Reply:
There's two ways to open a specific test App:
1. Open the link Directly, e.g., `https://0.0.0.0:9000/simpel-app`
   * It doesn't work for `localhost:8080/simple-app`, I demoed this to you before, there's a 403 error. For localhost URL we have to click the button to navigate. I think the issue here is about the webpack config. I use the same webpack config as our main demo. Once we fix that, we can enable opening the specific App directly for localhost URL.
2. Open the home page and click the button.

I don't think it hurts anything, it's good to keep both to provide flexibility. And we are likely to use this App to reproduce some issue in the future, having the menu makes things easier.